includes/rest/class-outbox-controller.php

```diff
@@ -120,8 +120,7 @@ public function get_items( $request ) {
 			'author'         => $user_id > 0 ? $user_id : null,
 			'paged'          => $page,
 			'post_type'      => Outbox::POST_TYPE,
-			'post_status'    => 'draft',
-
+			'post_status'    => 'any',
```

Comment:
```suggestion
			'post_status'    => 'any',

```
You don't want to keep the new line?